Extensions/messaging_tweaks.js

```diff
@@ -147,6 +152,32 @@ XKit.extensions.messaging_tweaks = new Object({
 				$(this).addClass("xkit-others_messaging_icon");
 				$(this).parents(".conversation-message").addClass("xkit-others_messaging_message");
 			}
+			if (XKit.extensions.messaging_tweaks.preferences.allow_emojis.value) {
+				var msg_div = $(this).parents(".conversation-message").find(".message-bubble .message");
+				
+				// Use regex to find emoji patterns in the chat message
+				// Find the string between the two ":" to find which image to use
+				// but only if there are emoji tags found
+				var emojis;
+				if (emojis = msg_div.text().match(/:(.*?):/gi)) {
```

Comment:
Linting fails on this (just move it out of the if statement and use `if (emojis)` or `if (emojis.length > 0)`)